marble/premain/premain.go

```diff
@@ -52,14 +52,17 @@ func readUUID(appFs afero.Fs, filename string) (*uuid.UUID, error) {
 
 	marbleUUID := uuid.New()
 	if err := marbleUUID.UnmarshalBinary(uuidBytes); err != nil {
-		return nil, fmt.Errorf("failed to unmarshal UUID: %v", err)
+		// try again assuming UUID was stored in text format
```

Comment:
Can we always marshal the uuid as a string, so we don't get the ambiguous case here?